jobs/stackdriver-nozzle/templates/stackdriver-nozzle-ctl.erb

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e
+
+RUN_DIR=/var/vcap/sys/run/stackdriver-nozzle
+LOG_DIR=/var/vcap/sys/log/stackdriver-nozzle
+PIDFILE=$RUN_DIR/stackdriver-nozzle.pid
+
+case $1 in
+
+start)
+
+mkdir -p $RUN_DIR
+mkdir -p $LOG_DIR
+
+chown -R vcap:vcap $LOG_DIR
+
+(crontab -l | sed /stackdriver.*logrotate/d; cat /var/vcap/jobs/stackdriver-nozzle/config/stackdriver_logrotate.cron) | sed /^$/d | crontab
+
+exec >>$LOG_DIR/stackdriver-nozzle.stdout.log \
+    2>>$LOG_DIR/stackdriver-nozzle.stderr.log
+
+
+export API_ENDPOINT=<%= p('firehose.api_endpoint') %>
+export FIREHOSE_USERNAME=<%= p('firehose.username') %>
+export FIREHOSE_PASSWORD=<%= p('firehose.password') %>
+export SKIP_SSL_VALIDATION=<%= p('firehose.skip_ssl_validation', false) %>
+
+export PROJECT_ID=<%= p('gcp.project_id') %>
+
+chpst -u vcap:vcap /var/vcap/packages/stackdriver-nozzle/bin/stackdriver-nozzle &
```

Comment:
Most of the cf managed processes use a `pid_guard` helper to ensure the process doesn't launch more than once. See
* https://github.com/cloudfoundry/cf-release/blob/master/src/common/utils.sh
* https://github.com/concourse/concourse/blob/master/jobs/tsa/templates/tsa_ctl.erb#L39